reference/orders.v2.oas2.yml

```diff
@@ -1940,6 +1940,7 @@ components:
                       display_style: Pick list
                   configurable_fields: []
                   gift_certificate_id: null
+                  discounted_total_inc_tax: '0.0000'
```

Comment:
🍹 should we provide the correct value here instead of 0.0000 given we have value for total_ex_tax, tax and discount ?